platform/packages/platform/src/bank.rs

```diff
@@ -576,9 +575,9 @@ mod test {
     #[derive(Clone)]
     struct Cmd<G>
     where
-        G: Group,
+        G: 'static + Group,
```

Comment:
[nit] For consistency with the rest of the code base, let's keep the lifetimes in the end.